androidApp/src/main/java/com/example/tasky/android/login/viewmodel/LoginViewModel.kt

```diff
@@ -53,8 +52,9 @@ class LoginViewModel(
 
         viewModelScope.launch {
             val isLoginSuccess =
-                loginManager.logIn(
-                    LoginBody(email = _screenStateFlow.value.emailState.text, password = _screenStateFlow.value.passwordState.text),
+                loginManager.login(
+                    email = _screenStateFlow.value.emailState.text,
+                    password = _screenStateFlow.value.passwordState.text,
```

Comment:
What's the difference between your `LoginManager` and `LoginRepository`? In this VM you use the manager and in the register VM the repository 🤔